app/controllers/dashboard_controller.rb

```diff
@@ -589,7 +589,7 @@ def logout
   def change_group
     # Get the user and new group and set current_group in the user record
     db_user = current_user
-    db_user.update(:current_group => MiqGroup.find_by(:id => params[:to_group]))
+    db_user.update(:current_group => db_user.miq_groups.find_by!(:id => params[:to_group]))
```

Comment:
yay for `!` :)

(currently one has to update users' `miq_groups` for `current_group` to become non-nil if that happens :))